SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -829,6 +843,12 @@ - (void)videoServiceProtocolDidUpdateFromOldProtocol:(nullable SDLProtocol *)old
     }
 }
 
+#pragma mark - Lifecycle Destruction
+
+- (void)dealloc {
+    [self.backgroundTaskManager endBackgroundTask];
```

Comment:
I think this can just be moved to the `backgroundTaskManager` dealloc.

Reply:
Fixed